src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/stats/FlowStatSpec.groovy

```diff
@@ -428,4 +430,52 @@ class FlowStatSpec extends HealthCheckSpecification {
         cleanup:
         flow && flowHelperV2.deleteFlow(flow.flowId)
     }
+
+    @Tidy
+    def "System is able to collect stats after partial updating(port) on a flow endpoint"() {
+        given: "Two active neighboring switches connected to the traffgens"
+        def traffGenSwitches = topology.activeTraffGens*.switchConnected*.dpId
+        def switchPair = topologyHelper.getAllNeighboringSwitchPairs().find {
+            it.src.dpId in traffGenSwitches && it.dst.dpId in traffGenSwitches
+        } ?: assumeTrue(false, "No suiting switches found")
+
+        and: "A flow with updated port on src endpoint via partial update"
+        def traffgenPortOnSrcSw = topology.activeTraffGens.find { it.switchConnected ==  switchPair.src}.switchPort
+        def srcFlowPort = (topology.getAllowedPortsForSwitch(topology.activeSwitches.find {
+            it.dpId == switchPair.src.dpId
+        }) - traffgenPortOnSrcSw).last()
+
+        def flow = flowHelperV2.randomFlow(switchPair).tap { it.source.portNumber = srcFlowPort }
+        flowHelperV2.addFlow(flow)
+
+        flowHelperV2.partialUpdate(flow.flowId, new FlowPatchV2().tap {
+            source = new FlowPatchEndpoint().tap {portNumber = traffgenPortOnSrcSw }
+        })
+
+        when: "Generate traffic on the flow"
+        Date startTime = new Date()
+        def traffExam = traffExamProvider.get()
+        def exam = new FlowTrafficExamBuilder(topology, traffExam).buildExam(northbound.getFlow(flow.flowId),
+                (int) flow.maximumBandwidth, 5)
+        exam.setResources(traffExam.startExam(exam, true))
+        assert traffExam.waitExam(exam).hasTraffic()
+
+        then: "System collects stats for ingress/egress cookies"
+        def flowInfo = database.getFlow(flow.flowId)
+        def tags = [switchid: switchPair.src.dpId.toOtsdFormat(), flowid: flow.flowId]
+        def metric = metricPrefix + "flow.raw.bytes"
+        def waitInterval = 10
```

Comment:
can be way smaller. Not an impossible request for otsdb, but 10s wait is very significant for test duration

Reply:
`10s` was chosen just for reducing the amount of requests to opentsdb.
changed to `5s`